modules/util/JitsiLocalStorage.js

```diff
@@ -37,7 +42,11 @@ class JitsiLocalStorage extends DummyLocalStorage {
         let storage;
 
         try {
-            storage = window.localStorage;
+            if (this.isIFrame()) {
+                storage = top.localStorage;
```

Comment:
Why would you want to do this? Basically this will attempt to use the localStorage of the parent page isn't it?

I think the try catch would gracefully handle the error. Isn't it am I missing something?

Reply:
This is when the page works as iframe but it doesn't work...

I know, I thought it may take the parent localStorage and replace it with your localStorage page but that not solve the problem.